recipe/build.sh

```diff
@@ -7,6 +7,7 @@ mkdir build && cd build
 cmake ${CMAKE_ARGS} .. -GNinja \
   -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
   -DCMAKE_INSTALL_LIBDIR=lib \
+  -DCMAKE_MODULE_PATH="${PREFIX}/lib/cmake" \
   -DBUILD_ONLY='s3;core;transfer;config;identity-management;sts' \
   -DENABLE_UNITY_BUILD=ON \
   -DENABLE_TESTING=OFF \
```

Comment:
Should we also disable `USE_OPENSSL` https://github.com/aws/aws-sdk-cpp/blob/1.10.55/Docs/CMake_Parameters.md#use_openssl and use our openssl?